Earthfile

```diff
@@ -1,7 +1,7 @@
 VERSION 0.8
 
-IMPORT github.com/input-output-hk/catalyst-ci/earthly/mdlint:v3.4.9 AS mdlint-ci
-IMPORT github.com/input-output-hk/catalyst-ci/earthly/cspell:v3.4.9 AS cspell-ci
+IMPORT github.com/input-output-hk/catalyst-ci/earthly/mdlint:v3.5.3 AS mdlint-ci
```

Comment:
can you make this 3.5.4? 

https://github.com/input-output-hk/catalyst-ci/releases/tag/3.5.4